### PR TITLE
Remove alternate editors, old themes, and unused javascript

### PR DESCRIPTION
**Pull request by drgrice1**
Get rid of old junk.

The first commit removes the alternate and mostly unused editors (UserList.pm, ProblemSetDetail.pm, SetMaker?.pm, ProblemSetDetail.pm).  The currently used versions are renamed without the numerical suffix.

The second commit removes deprecated themes and unused javascript.

### CODE REVIEW

**Comment by pstaabp** [comment]
Trying to navigate through all of the pages:

- `node instructor_set_maker2 doesn't exist in node list: failed at /Users/pstaab/code/ww-docker/webwork2/lib/WeBWorK/URLPath.pm line 1089.` on the Statistics, Student Progress, Scoring Tools, Email and Course Config  pages.

File: lib/WeBWorK/URLPath.pm [hunk 1]
```diff
@@ -332,8 +319,9 @@ our %pathTypes = (
...
 		name    => x('Instructor Tools'),
 		parent  => 'set_list',
-		kids    => [ qw/instructor_user_list instructor_user_list2 instructor_set_list instructor_set_list2
-		    instructor_add_users instructor_achievement_list 
+		kids    => [ qw/
+			instructor_user_list instructor_set_list
+			instructor_add_users instructor_achievement_list 
 			instructor_set_assigner instructor_file_manager
 			instructor_problem_editor
 			instructor_set_maker instructor_set_maker2 instructor_set_maker3 instructor_set_maker_no_js
```
Comment (pstaabp) [inline lib/WeBWorK/URLPath.pm:327]: Looks like this is where the problem is.

**Comment by drdrew42** [comment]
Oh, I love trash!

**Review by drdrew42 — APPROVED** [review]

**Review by pstaabp — APPROVED** [review]
Looks like this looks good now.

**Pull request merged**